share/examples/generateExampleDCP.py

```diff
@@ -0,0 +1,75 @@
+# Run this file to create an example DCP dataset
+
+
+from netCDF4 import Dataset
+import numpy as np
+
+
+N = 200
```

Comment:
Can we add command line arguments to control the number of points and time steps? Optionally outputting velocities would also be useful.

Reply:
This script is designed to show users how to output DCP data. It is very heavily documented and designed to be edited by the end user. I think it would be reasonable that if someone, for some reason, wants to customize the example dataset they just edit the file. Also, this script is designed to be as simple and self-explanatory as possible and adding a bunch of argument-handling will, I believe, complicate it unnecessarily. If you insist, however, I will add them.